system_config.yaml

```diff
@@ -34,7 +34,7 @@ ANSWER_MULT_QS_FORMAT_RESP_PROMPT: "format_mult_qs_prompt.jinja"
 PATH_TO_DOMAIN_CONFIGS: "domain_specific/configs/restaurant_configs"
 MODEL: "gpt-3.5-turbo"
 SEARCH_ENGINE: "vector database"
-ENABLE_MULTITHREADING: True
+ENABLE_MULTITHREADING: False
 UNACCEPTABLE_SIMILARITY_SCORE_RANGE: 0.5
```

Comment:
is there a reason why you changed this? if not, just set it to True

Reply:
Oh, sorry. That was my mistake, I'll fix it.